nox/virtualenv.py

```diff
@@ -104,13 +104,13 @@ def locate_using_path_and_version(version):
     if not version:
         return None
 
+    script = "import platform; print(platform.python_version())"
```

Comment:
Why lift this out of the `if` statement but not out of the function as a module global?

Reply:
I was aiming for style consistency with `locate_via_py`. I can put it inside the `if` block if it helps (would prefer not to make it module global, as it is the only time this string is used). Should I?